index.html

```diff
@@ -234,5 +234,63 @@ <h2>Contact me</h2>
      </div>
     </footer> 
     <script src="script.js"></script> 
+    <div class="container-modal">
+      <div class="inner-container">
+      <div class = "main-container-head">
+      <h2>Tonic</h2>
+      <img class = "popup-cross" src="./images/Enabled-3.svg" alt="X">
+      </div>
```

Comment:
- [ ] Please make sure that the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for projects list should be removed, and instead it should be added via JavaScript.** Check an example of students projects: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95